tests/ui/feature-gates/feature-gate-repr-scalable.rs

```diff
@@ -0,0 +1,8 @@
+#![feature(repr_simd)]
+
+#[repr(simd, scalable(16))] //~ error: Scalable SIMD types are experimental
+struct Foo {
+    _ty: [i8; 0],
```

Comment:
This type is a lie and will give issues for `Foo { _ty: [] }`. Maybe use `_ty: [i8]` instead? That will also automatically mark it as `!Sized`.